054-AzureLoadTesting/Coach/Solution-07.md

```diff
@@ -0,0 +1,37 @@
+# Challenge 07 - Load Testing With Chaos Experiment (Resilience Testing) - Coach's Guide 
+
+[< Previous Solution](./Solution-06.md) - **[Home](./README.md)**
+
+## Notes & Guidance
+
+Here are instructions on how to setup Chaos Studio and run the experiment.  Chaos studio does not offer a GitHub action at this time.  We want the users to get a taste of chaos engineering so they can trigger the chaos experiment manually while the load test runs.  They are not required to implement this all in the pipeline.  That is an advanced challenge.
```

Comment:
capitalize Chaos Studio & GitHub Actions, do this throughout

Reply:
Done